### DISCUSSION TITLE
[Feature Request] Amazon Textract API

### DISCUSSION THREAD

**Discussion by gburgett** [supabase/supabase#14299] [Feature Requests]
Hello!

For my app, I have a need to run [Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/invoices-receipts.html) against files uploaded by my users to Supabase Storage.  Amazon Textract can be invoked on documents uploaded to S3 buckets (see doc here: https://docs.aws.amazon.com/textract/latest/dg/sync-calling.html)

Is there a way I can grant Amazon Textract to access the files uploaded to Supabase Storage?  Perhaps by generating some kind of signed ID?

**Comment by jh0274** [comment]
this would be great! i need the same thing

**Comment by milanb-crew** [comment]
also interested in this functionality!

**Comment by michedapt** [comment]
I would also be interested in this! I have this same problem.

**Comment by john-dw** [comment]
I too !

**Comment by aeynaud** [comment]
Would be awesome! I would love to use this. Please let's have a bump for this one :)

**Comment by matrinox** [comment]
I have a feeling they can't release this because it's something AWS would have to build out on their end
